app/controllers/machine.js

```diff
@@ -76,7 +76,7 @@ function getBackupEvents(machine, CALENDAR_ROWS = 5)
       id: undefined,
       bucket: false
     });
-  }
+  }  
```

Comment:
Here, as well as in `app/views/layout.njk`, you make changes to whitespace without changing anything else. You should be able to configure your editor to prevent this behavior, but whether you do that or not, it's a good idea to review staged changes (`git diff --cached`) before committing anything. Please revert these whitespace changes back to how they were on `devel`.